tests/features/interactive_steps.feature

```diff
@@ -145,4 +145,19 @@ Scenario: Base64 encoded corner cases are decoded correctly
     | object_radio_field | object_radio_choice_one | |
     | object_checkboxes_test['obj_chkbx_opt_1'] | True | |
     | object_dropdown | obj_opt_2 | |
-    | checkbox_dict['single_quote_key']['🁳🧴🯍🥫🅍🗎🹇🗷🌲🯵'] | True | |
\ No newline at end of file
+    | checkbox_dict['single_quote_key']['🁳🧴🯍🥫🅍🗎🹇🗷🌲🯵'] | True | |
+
+@fast @i6
+Scenario: Doesn't try to 'continue' with a restart button
```

Comment:
TBH the main reason the tests is there is to avoid an infinite loop, not to particularly test that the failure occurs, which IMO makes it different from the other failure tests, i.e. if there was a way for the test to succeed instead, I would do that. But I guess it's better to match the convention, and just explain in the comments as you did.